### PR TITLE
Surface menu details to menu list

### PR DESCRIPTION
**Pull request by iisa**
When a menu has extra details, add surface them on the menu list

ezgif com-video-to-gif (10)

### CODE REVIEW

File: src/ia-menu-slider.js [hunk 1]
```diff
@@ -87,20 +88,22 @@ export class IAMenuSlider extends LitElement {
...
         <div class="details">
-          <h3>${title}</h3>
+          <h3>${label}</h3>
           <span class="extra-details">${menuDetails}</span>
         </div>
         ${actionSection}
         <button class="close" aria-label="Close this menu" @click=${this.closeMenu}>
-          <ia-icon-collapse-sidebar></ia-icon-collapse-sidebar>
+          <ia-icon-collapse-sidebar
+            style="--iconWidth: var(--closeIconWidth); --iconHeight: var(--closeIconHeight);"
```
Comment (shaneriley) [inline src/ia-menu-slider.js:105]: These styles can be moved to src/styles/menu-slider.js, like:
```css
  ia-icon-collapse-sidebar {
    --iconWidth: var(--closeIconWidth);
    --iconHeight: var(--closeIconHeight);
  }
```

File: src/styles/menu-button.js [hunk 2]
```diff
@@ -18,6 +18,10 @@ export default css`
     align-items: center;
   }
 
+  button.menu-item {
+    border: none;
```
Comment (shaneriley) [inline src/styles/menu-button.js:22]: You can just move this into the class selector block above. Both elements should have identical styles when rendered. With the addition of `button` as a possibility, these properties also need to be added:

```css
  cursor: pointer;
  -webkit-appearance: none;
  appearance: none;
  border-radius: 0;
```

**Review by shaneriley — CHANGES_REQUESTED** [review]

File: src/ia-menu-slider.js (hunk 1 above)
Comment (iisa) [reply src/ia-menu-slider.js:105]: done

File: src/styles/menu-button.js (hunk 2 above)
Comment (iisa) [reply src/styles/menu-button.js]: thanks, just updated and also moved cursor up

**Comment by iisa** [comment]
updated to address code review from @shaneriley . will merge.

**Pull request merged**